packages/core/src/registries/CustomStyleRegistry.ts

```diff
@@ -15,8 +16,10 @@ export abstract class CustomStyleRegistry {
    * @returns {void}
    */
   public static define(name: string, css: string): void {
-    if (register.has(name)) {
+    if (register.has(name) && isLocalhost) {
       throw new DuplicateStyleError(name);
+    } else if (register.has(name) && !isLocalhost) {
```

Comment:
There is no need to check `register.has(name)` twice.

Reply:
Fixed